infra/feast-operator/internal/controller/services/services_types.go

```diff
@@ -172,22 +177,25 @@ type RepoConfig struct {
 
 // OfflineStoreConfig is the configuration that relates to reading from and writing to the Feast offline store.
 type OfflineStoreConfig struct {
-	Host string            `yaml:"host,omitempty"`
-	Type OfflineConfigType `yaml:"type,omitempty"`
-	Port int               `yaml:"port,omitempty"`
+	Host         string                 `yaml:"host,omitempty"`
+	Type         OfflineConfigType      `yaml:"type,omitempty"`
+	Port         int                    `yaml:"port,omitempty"`
+	DBParameters map[string]interface{} `yaml:",inline,omitempty"`
 }
 
 // OnlineStoreConfig is the configuration that relates to reading from and writing to the Feast online store.
 type OnlineStoreConfig struct {
-	Path string           `yaml:"path,omitempty"`
-	Type OnlineConfigType `yaml:"type,omitempty"`
+	Path         string                 `yaml:"path,omitempty"`
+	Type         OnlineConfigType       `yaml:"type,omitempty"`
+	DBParameters map[string]interface{} `yaml:",inline,omitempty"`
 }
 
 // RegistryConfig is the configuration that relates to reading from and writing to the Feast registry.
 type RegistryConfig struct {
-	Path               string             `yaml:"path,omitempty"`
-	RegistryType       RegistryConfigType `yaml:"registry_type,omitempty"`
-	S3AdditionalKwargs *map[string]string `json:"s3_additional_kwargs,omitempty"`
+	Path               string                 `yaml:"path,omitempty"`
+	RegistryType       RegistryConfigType     `yaml:"registry_type,omitempty"`
+	S3AdditionalKwargs *map[string]string     `json:"s3_additional_kwargs,omitempty"`
```

Comment:
can you fix this? its not related to your change but needs to be fixed.
```suggestion
	S3AdditionalKwargs *map[string]string     `yaml:"s3_additional_kwargs,omitempty"`
```

Reply:
Done